.github/workflows/updateVersionsAndCreatePR.yml

```diff
@@ -0,0 +1,85 @@
+name: Update Versions and Create PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version (x.x.x)'
+        required: true
+      zowe_version:
+        description: "Zowe Version Tag"
+        default: zowe-v2-lts
+        required: true
+        type: choice
+        options:
+        - zowe-v1-lts
+        - zowe-v2-lts
+
+jobs:
+  update_versions_and_create_pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update -y
+          sudo apt install gh

```

Comment:
`sudo apt install gh` can be removed as GitHub CLI is packaged w/ the `ubuntu-latest` image: https://github.com/actions/runner-images/blob/8077d7b42b297abc90177489c58aec67f0d5c1f2/images/linux/Ubuntu2204-Readme.md#cli-tools